src/helpers/moderation.ts

```diff
@@ -42,10 +42,15 @@ export function setData(result?: Record<string, string[]>) {
 }
 
 export default async function run() {
-  setData(await loadModerationData());
+  while (true) {
+    try {
+      setData(await loadModerationData());
+    } catch (e: any) {
+      capture(e);
+    }
```

Comment:
Already have a trycatch inside `loadModerationData` which capture the error , can similify to
```ts
export default async function run() {
  while (true) {
    setData(await loadModerationData());
    await snapshot.utils.sleep(20e3);
  }
}
```